[2021-10-02 04:10:37,114] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-02T03:59:45.116659+00:00 [queued]>
[2021-10-02 04:10:37,211] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-02T03:59:45.116659+00:00 [queued]>
[2021-10-02 04:10:37,213] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:37,232] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-02 04:10:37,234] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:37,288] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_A> on 2021-10-02T03:59:45.116659+00:00
[2021-10-02 04:10:37,315] {standard_task_runner.py:52} INFO - Started process 17334 to run task
[2021-10-02 04:10:37,351] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_A', '2021-10-02T03:59:45.116659+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmps__9nv0x', '--error-file', '/tmp/tmpvno15yna']
[2021-10-02 04:10:37,362] {standard_task_runner.py:77} INFO - Job 8: Subtask Training_Model_A
[2021-10-02 04:10:37,605] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_A 2021-10-02T03:59:45.116659+00:00 [running]> on host d6b4131456ad
[2021-10-02 04:10:38,057] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T03:59:45.116659+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-02T03:59:45.116659+00:00
[2021-10-02 04:10:38,059] {python.py:151} INFO - Done. Returned value was: 2
[2021-10-02 04:10:38,252] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_A, execution_date=20211002T035945, start_date=20211002T041037, end_date=20211002T041038
[2021-10-02 04:10:38,404] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-02 04:10:38,648] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
